website/backend/publications/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
```

Comment:
in the near future, we shall all be required to have unit tests for all our implementations. Glad that we already have it in consideration